### PR TITLE
Add pyobjc-framework-coreservices

### PR DESCRIPTION
**Pull request by andfoy**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by andfoy** [comment]
@steff456 @ccordoba12 @dalthviz Do you agree with being maintainers of this feedstock?

File: recipes/pyobjc-framework-coreservices/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+build:
+  skip: true   # [not osx]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```
Comment (carterbox) [inline recipes/pyobjc-framework-coreservices/meta.yaml:22]:
```suggestion
```
I think the cxx compiler is not needed, and the link inspector is complaining about overdepending on the cxx standard library.

File: recipes/pyobjc-framework-coreservices/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pyobjc-core {{ version }}
+    - pyobjc-framework-fsevents {{ version }}
```
Comment (carterbox) [inline recipes/pyobjc-framework-coreservices/meta.yaml:29]:
```suggestion
    - pyobjc-core >={{ version }}
    - pyobjc-framework-fsevents >={{ version }}
```
The upstream version spec is >= not ==.

https://github.com/ronaldoussoren/pyobjc/blob/26f2a5dfb8a52baf851b2890267e27a916b1f3f8/pyobjc-framework-CoreServices/setup.py#L35

**Review by carterbox — CHANGES_REQUESTED** [review]

**Comment by carterbox** [comment]
All maintainers except @steff456 have indicated.

**Comment by steff456** [comment]
I agree!

**Pull request merged**